css/layout/layout.css

```diff
@@ -23,3 +22,28 @@
     max-width: 2280px;
   }
 }
+
+section {
```

Comment:
as I said before, even if this is easier, eventually you will have to change it because of different UX/UI strategies